src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -93,11 +97,15 @@ public ReportParser(Configuration configuration) {
             throw new ValidationException(e);
         }
     }
-
-    /** Sets additional information and calculates values which should be calculated during object creation. */
-    private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
-        for (Feature feature : features) {
+    
+    
+    /** 
+     * Collects features, sets additional information and calculates values which should be calculated during object creation. 
+     */
+    private void collectFeaturesAndSetMetadata(Collection<Feature> featuresAccumulator, Feature[] features, String jsonFile, int jsonFileNo) {
+    	for (Feature feature : features) {
```

Comment:
if you remove tab from this line probably there won't be difference as lines before and after are the same